api/proto/common/v2/common_v2.proto

```diff
@@ -60,10 +60,10 @@ message Batch {
   repeated BlobCertificate blob_certificates = 2;
 }
 
-// PaymentHeader contains payment information for a blob.
-// At least one of reservation_period or cumulative_payment must be set, and reservation_period 
-// is always considered before cumulative_payment. If reservation_period is set but not valid, 
-// the server will reject the request and not proceed with dispersal. If reservation_period is not set 
+// PaymentHeader contains payment information for a blob. 
+// At least one of timestamp or cumulative_payment must be set, and timestamp 
+// is always checked before cumulative_payment. If timestamp is set but not valid, 
+// the server will reject the request and not proceed with dispersal. If timestamp is not set 
```

Comment:
Is the content below still up-to-date?

Reply:
mostly up-to-date but I decided to rewrite some parts to make it more clear